test_aeo/dsgrid_project/datasets/benchmark/Commercial_End_Use_Growth_Factors/dataset.toml

```diff
@@ -98,6 +105,7 @@ Weather years, 2013-2050, trivial, single record
 [[dimensions]] # already exists
 type = "model_year"
 name = "Model_Years-trivial-2013_to_2050"
+display_name = "None"
```

Comment:
So users are supposed to put "None" in for all trivial dimensions?

Reply:
Uhh...I think you made that commit.